feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.ts

```diff
@@ -683,4 +684,16 @@ export class ConfiguratorGroupMenuComponent {
   isDialogActive(configuration: Configurator.Configuration): boolean {
     return configuration.interactionState.showConflictSolverDialog ?? false;
   }
+
+  /**
```

Comment:
Do we need any JS doc for this method? 
If `yes`, please, explain what the method intends to do.
It is not clear, for instance:
> If the feature toggle s enabled then the group ID will be returned, otherwise the group itself.